specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2022-10-01/skus.json

```diff
@@ -0,0 +1,275 @@
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PublisherNameParameter"
+          },
+          {
+            "$ref": "#/parameters/OfferNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SkuNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ODataExpandParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Sku"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Sku": {
+            "$ref": "./examples/GetSku.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SkuList": {
+      "description": "List of SKU proxy resources for the HCI cluster.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of SKU proxy resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Sku"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Sku": {
+      "description": "Sku details.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "SKU properties.",
+          "$ref": "#/definitions/SkuProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "SkuProperties": {
+      "description": "SKU properties.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning State",
+          "type": "string",
+          "readOnly": true
+        },
+        "publisherId": {
+          "type": "string",
+          "description": "Identifier of the Publisher for the offer"
+        },
+        "offerId": {
```

Comment:
>offerId [](http://example.com/codeflow?start=9&length=7)

Same here and for catalogPlanId, marketplaceSkuId.

Reply:
Will not fix